stories/examples/Ratings.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
+import { RatingBar }  from 'react-ui-core/src'
+import { Square, Circle } from '../SampleRatingComp'
+import { RatingBarTheme } from '../theme'
+
+export const DefaultRatings = (
+  <RatingBar
+    ratingScale={5}
+    rating={2.35}
+    // total= {'20 Ratings'}
+    Color={'royalblue'}
```

Comment:
Please change these colors to just a string:

```
color="royalBlue"
```

The rest below need changed as well.